book/src/installation-binaries.md

```diff
@@ -0,0 +1,37 @@
+# Pre-built Binaries
+
+Each Lighthouse release contains several downloadable binaries in the "Assets"
+section of the release. You can find the [releases
+on Github](https://github.com/sigp/lighthouse/releases).
+
+> Note: binaries are not yet provided for MacOS or Windows native.
+
+## Platforms
+
+Binaries are supplied for two platforms:
+
+- `x86_64-unknown-linux-gnu`: AMD/Intel 64-bit processors (most desktops, laptops, servers)
+- `aarch64-unknown-linux-gnu`: 64bit ARM processors (Raspberry Pi 4)
+
+Additionally there is also a `-portable` suffix which indicates if the `portable` feature is used:
+
+- Without `portable`: uses modern CPU instructions to provide the fastest signature verification times (may cause `Illegal instruction` error on older CPUs)
+- With `portable`: approx. 20% slower, but should work on all modern 64-bit processors.
+
+## Usage
+
+Each binary is contained in a `.tar.gz` archive. For this example, lets use the
+`v0.2.13` release and assume the user needs a portable `x64_64` binary.
+
+> Whilst this example uses `v0.2.13` we recommend always using the latest release.
+
+### Steps
+
+1. Go to the [`v0.2.13` release](https://github.com/sigp/lighthouse/releases/tag/v0.2.13).
```

Comment:
I think linking to a specific release tag could be confusing and I don't think we lose too much by having something like `lighthouse-${VERSION}-x86_64-unknown-linux-gnu-portable.tar.gz`